### ISSUE TITLE
[1.3.0-RC3] Cannot import sbt project in IntelliJ IDEA

### ISSUE THREAD

**Issue by plokhotnyuk** [sbt/sbt#4880]
## steps

Prerequisites: sbt 1.3.0-RC3, Ubuntu 18.04, Intellij IDEA 2019.1.3 + the latest Scala language plugin

1. Clone repo: https://github.com/plokhotnyuk/rtree2d
2. Try to import it as IntelliJ IDEA project:
```
File > New > Project from Existing Sources...`
```
## problem

```
[info] sbt server started at local:///home/andriy/.sbt/1.0/server/a0381557a097f294b454/sock
sbt:rtree2d>
[error] Expected ';'
[error] Not a valid key: dumpStructure (similar: buildStructure)
[error] */*:dumpStructure
[error]                  ^
sbt:rtree2d>
[info] shutting down server
```

## expectation

Succesfuly imported project

## notes

W/A is rollback to sbt 1.3.0-RC2

**Comment by dejan2609** [comment]
I believe this is solved here #4875 and here #4874

**Comment by plokhotnyuk** [comment]
@dejan2609 thanks! closing and waiting for new release...
